shinken/objects/host.py

```diff
@@ -874,6 +881,31 @@ def add_host_chk_dependency(self, h, status, timeperiod, inherits_parent):
     def add_service_link(self, service):
         self.services.append(service)
 
+
+    def __repr__(self):
+        return '<Host host_name=%r name=%r use=%r />' % (
+            getattr(self, 'host_name', None),
+            getattr(self, 'name', None),
+            getattr(self, 'use', None))
```

Comment:
This is not part of the pull request right? I mean, it's only to get fancy oupout not a technical need for the feature.